layouts/partials/header.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
 <html>
-    <head>
+    <head prefix="og: http://ogp.me/ns# fb: http://ogp.me/ns/fb# article: http://ogp.me/ns/article#">
```

Comment:
1. Well that error is shown when you do not provide the image for a post. If you set image for your post/page, we will display it:
https://github.com/jbub/ghostwriter/blob/master/layouts/partials/opengraph.html#L5-L9
2. I have just read about the prefix attribute and it looks like we do not need it (yet) as per: 
https://www.w3.org/TR/xhtml-rdfa-primer/#h-note8